src/config/chis-tg/config.json

```diff
@@ -0,0 +1,189 @@
+
+{
+  "domains": [
+    {
+      "friendly": "MoH Togo DEV",
+      "domain": "moh-togo.dev.medicmobile.org"
+    },
+    {
+      "friendly": "MoH Togo UAT",
+      "domain": "moh-togo-uat.dev.medicmobile.org"
+    }
+  ],
+  "contact_types": [
+    {
+      "name": "cg70_supervision",
+      "friendly": "Supervision Area",
+      "contact_type": "cg72_supervision",
+      "user_role": ["supervisor"],
+      "username_from_place": false,
+      "deactivate_users_on_replace": true,
```

Comment:
Note you should have a deactivated role [like this](https://github.com/medic/config-ug-vht/blob/master/app_settings/base_settings.json#L192) in your app if you're going to set this to true.